website_sale_product_attachment/migrations/13.0.2.0.0/pre-migration.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2021 Tecnativa - Pedro M. Baeza
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+from odoo import SUPERUSER_ID, api
+
+
+def migrate(cr, version):
+    """Remove offending view for allowing to update."""
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    view = env.ref("website_sale_product_attachment.download_icons", False)
+    view.unlink()
```

Comment:
This will fail if not found:

```suggestion
    if view:
        view.unlink()
```